google-cloud-bigquery/lib/google/cloud/bigquery.rb

```diff
@@ -186,7 +186,8 @@ module Cloud
     # | `BOOL`      | `true`/`false` | |
     # | `INT64`     | `Integer`      | |
     # | `FLOAT64`   | `Float`        | |
-    # | `STRING`    | `STRING`       | |
+    # | `NUMERIC`   | `BigDecimal`   | Will be rounded to 9 decimal places |
```

Comment:
I think I would use the term "scale" here:

Will be rounded if necessary to fit 9 decimal digits of scale.

It might also be good to state the rounding policy.

Reply:
I actually find "precision" and "scale" more confusing than "decimal places". We have very little space here, which is why I simplified the wording.